src/main/java/pageObjects/Licenses.java

```diff
@@ -36,173 +33,149 @@ public Licenses(WebDriver driver) {
     private WebElement licenseLocationList;
     @FindBy(id = "license-locations-selectList-romania")
     private WebElement RomaniaList;
-    @FindBy(id = "license_dropdown_1111")
+    @FindBy(id = "license_dropdown_135")
     private WebElement licenceDropdown;
-    @FindBy(id = "license_edit_1111")
+    @FindBy(id = "license_edit_135")
```

Comment:
I would recommend to try to not hardcode this, I think you can watch how was made on the page with Devices.